reports/p2pstats.py

```diff
@@ -0,0 +1,160 @@
+import csv
+import numpy as np
+from collections import Counter
+
+# Function to calculate mean, median, and standard deviation
+def calculate_stats(data):
+    mean = np.mean(data)
+    median = np.median(data)
+    std_dev = np.std(data)
```

Comment:
Note that these can be computed online, if we need to stream data; the algorithm is known as Welford's algorithm and [here (l23-49)](https://osdn.net/users/corbin/pf/cammy/scm/blobs/master/sampler/draw.py) is a Python 2.7 version which anybody may crib.